tests/e2e/README.md

```diff
@@ -88,7 +88,7 @@ E2E tests have multiple options available while running them as follows:
 * `--skip_setup` - Skip namespace creation and istio cluster setup (default: false)
 * `--skip_cleanup` - Skip the cleanup steps (default: false)
 * `--namespace <namespace>` - If you don't specify `namespace`, a random namespace is generated for each test.
-* `--use_local_cluster` - If true the tests will run within the cluster. If running on minikube, this should be set to true. (default: false)
+* `--use_local_cluster` - If true any loadbalance type services will be converted to a node port service during testing. If running on minikube, this should be set to true. (default: false)
```

Comment:
Just to make the term clearer per k8s doc:
loadbalance --> LoadBalancer
node port --> NodePort type